tools/cli/go-whisk-cli/commands/action.go

```diff
@@ -310,7 +311,7 @@ var actionListCmd = &cobra.Command{
         if actions, _, err = client.Actions.List(qualifiedName.entityName, options); err != nil {
             return actionListError(qualifiedName.entityName, options, err)
         }
-
+        
```

Comment:
I suggest using a prehook that runs our scan code - this will save you from failed travis builds: https://github.com/apache/incubator-openwhisk/wiki/Contributing:-Git-guidelines